src/utils/make-store.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react'
+
+const dispatchNoop: React.Dispatch<any> = (value: any) => {
+  return <></>
+}
+
+export default function makeStore(
```

Comment:
I tried the "context way" and it didn't work at all and flashed some layout

It was a challenge to get the types right here though so I want to look at it some more and committed it for posterity 😅

Reply:
inspired by [Cassidy Williams](https://twitter.com/cassidoo?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor) in [this livestream session on YouTube](https://www.youtube.com/watch?v=vrmpUCpmawg)